app/graphql/mutations/digital_object/update_projects.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+module Mutations
+  module DigitalObject
+    class UpdateProjects < Mutations::BaseMutation
+      argument :id, ID, required: true
+      argument :primary_project, Inputs::StringKey, required: true
+      argument(
+        :other_projects,
+        [Inputs::StringKey],
+        "An array representing the new set of other projects for the given object, "\
+        "or an empty array to remove all projects but the primary.",
+        required: true
```

Comment:
```suggestion
        required: false
```
Should this be `required: false` so users can use this mutation to update only the other_projects without having to supply primary_project?